testdata/buildkit-ssh/Dockerfile

```diff
@@ -0,0 +1,5 @@
+# syntax = docker/dockerfile:1.0-experimental
+FROM alpine
+
+# shows secret from custom secret location:
+RUN --mount=type=secret,id=my_ssh_key cat ./id_rsa_test
```

Comment:
Would be nice to use `type=ssh` to demonstrate that this works with the intended use case. Maybe it could just run `ssh-add -l`?

side note: since the test just runs the build and asserts no error, the `Dockerfile` will have to perform some sort of assertion itself to make sure SSH was configured correctly.

Reply:
I will go ahead to merge the PR and then fix the test